### PR TITLE
fix(islamic-website): index page markup fixes

### PR DESCRIPTION
**Pull request by hadzhehsen**
Closes: HQW-92

### CODE REVIEW

File: libs/islamic-website/index-page/src/lib/learn-and-grow-block/learn-and-grow-block.tsx [hunk 1]
```diff
@@ -165,7 +165,8 @@ export function LearnAndGrowBlock() {
           </Text>
           <div className="mt-[32px] w-full md:mt-[34px] md:w-fit">
             <Button className="w-full !px-[20px] md:!px-[32px]">
-              Get started with haqq wallet
+              Get started with <br className="block min-[400px]:hidden" /> haqq
+              wallet
```
Comment (olegshilov) [inline libs/islamic-website/index-page/src/lib/learn-and-grow-block/learn-and-grow-block.tsx]:
```suggestion
              Get started with HAQQ&nbsp;wallet
```

File: libs/islamic-website/index-page/src/lib/portfolio-block/portfolio-block.tsx [hunk 2]
```diff
@@ -32,7 +32,9 @@ export function PortfolioBlock() {
                 Manage Your <GradientText>Shariah Portfolio </GradientText>
                 with HAQQ Wallet
               </div>
-              <Text isMono>Ethics-First, Non-Custodial Wallet</Text>
+              <Text isMono className="lg:mt-[40px]">
+                Ethics-First, Non-Custodial Wallet
+              </Text>
```
Comment (olegshilov) [inline libs/islamic-website/index-page/src/lib/portfolio-block/portfolio-block.tsx]:
```suggestion
              <div className="lg:pt-[40px]">
                <Text isMono>
                  Ethics-First, Non-Custodial Wallet
                </Text>
              </div>
```

File: libs/islamic-website/ui-kit/src/lib/members-container/members-container.tsx [hunk 3]
```diff
@@ -87,7 +87,7 @@ export function MembersContainer({
           })}
         </Swiper>
       ) : (
-        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] md:grid-cols-4 lg:mt-[32px]">
+        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] lg:mt-[32px] lg:grid-cols-4">
```
Comment (olegshilov) [inline libs/islamic-website/ui-kit/src/lib/members-container/members-container.tsx:90]: There is not a single place in the layouts where the cards are 4 in a row

```suggestion
        <div className="mt-[24px] grid grid-cols-2 gap-[32px] md:mt-[28px] lg:mt-[32px] md:grid-cols-3">
```

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/islamic-website/index-page/src/lib/learn-and-grow-block/learn-and-grow-block.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/islamic-website/index-page/src/lib/learn-and-grow-block/learn-and-grow-block.tsx]: Resolved

File: libs/islamic-website/index-page/src/lib/portfolio-block/portfolio-block.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/islamic-website/index-page/src/lib/portfolio-block/portfolio-block.tsx]: Resolved

**Review by olegshilov — APPROVED** [review]

**Pull request merged**